dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue

```diff
@@ -87,7 +90,16 @@ export default {
     },
     scrollToTop(){
       window.scrollTo(0, 0);
-    }
+    },
+    goToData(event){
+      const company=event.data.companyId;
+      this.$router.push('/companies/' + company + '/eutaxonomies')
+    },
   },
 }
-</script>
\ No newline at end of file
+</script>
+<style>
```

Comment:
Dieses Style gilt dann leider globally - also in der ganzen Vue app - nicht nur in dieser Komponente.
Besser:
```
<style scoped>
```

Reply:
Über #-Zeichen für jeweiliges Element gescoped

Reply:
Ich habe den tag auf `<style scoped>` umgestellt.

Reply:
Wenn der style scoped ist, wird er nicht mehr auf die DataTable angewendet. Das war auch der Grund weshalb der style ohne scoped definiert wurde und über # den Elementen zugeorndet wurde. Ich schau aber mal ob man das mit style scoped funktionabel bekommt.

Reply:
verstanden - passt.